app/views/items/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Customer Details:</h1>
+
+<% @items.each do |item| %>
+    <li>Name: <%= item.name %></li>
+    <li>Address: <%= item.price %></li>
```

Comment:
Just make sure you update the text labels here to be price, supermarket, etc instead of address (but it's okay to reference syntax from other projects! Just be careful with copy/paste